apps/worker/src/app/workflow/usecases/subscriber-job-bound/subscriber-job-bound.usecase.ts

```diff
@@ -108,11 +104,6 @@ export class SubscriberJobBound {
 
     // If no subscriber makes no sense to try to create notification
     if (!subscriberProcessed) {
-      /**
```

Comment:
removing this one as we could potentially have a lot more comments like those, we should revisit the flow until the creation of the job once we won't be limited by the job entity, and add execution details in the way.